File: DSA/Basic Maths/gcd.cpp
#include <bits/stdc++.h>
using namespace std;
int gcd(int a, int b)
{
    return b == 0 ? a : gcd(b, a % b);   
}
int main()
{
    system("cls");
    cout<<gcd(12,8);
    return 0;
}

File: JAVA/Operators/Methods/src/GCD.java
public class GCD {
    static int GCD (int x, int y) {
        while (x!=y) {
            if (x > y) {
                x = x-y;
            }
            else
                y = y-x;
        }
        return x;
    }
    public static void main (String args[]) {
        System.out.println(GCD(225,15));
    }
}
